src/spring/azext_spring/tests/latest/test_asa_customized_accelerator.py

```diff
@@ -0,0 +1,50 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+
+from azure.cli.testsdk import (ScenarioTest, record_only)
+
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+
+@record_only()
```

Comment:
May I ask why those new tests are marked as `record_only()`? Could you add the reason to the comment to explain it?